tests/e2e/specs/Metrics2.spec.js

```diff
@@ -0,0 +1,392 @@
+        task: "Start Application",
+        email: "solutions+banker@processmaker.com"
+      }
+    ],
+    acceptedDocuments: null,
+    applicationConfig: {
+      UseIDAForWebLeads: true,
+      accountNumberSource: "InputAccountNumber",
+      UseAlloyForVerification: false,
+      UseExperianforVerification: true,
+      UseNeoCheckForVerification: true,
+      ExistingCustomerLookupSource: "Nxtsoft",
+      UseChexSystemsForVerification: true
+    },
+    applicationUpdated: "Application updated successfully",
+    applicationUpdateError: null,
+    applicationTrackerError: null
+  };
+
+  beforeEach(() => {
+    cy.server();
+    cy.visit("/");
+    cy.route(
+      "POST",
+      "/api/1.0/requests/data_sources/38",
+      JSON.stringify({
+        status: 200,
+        response: {
+          data: [],
+          meta: {
+            filter: "",
+            sort_by: "",
+            sort_order: "",
+            count: 2,
+            total_pages: 1,
+            current_page: 1,
+            from: 1,
+            last_page: 1,
+            path: "/api/1.0/collections/1/records",
+            per_page: 10,
+            to: 2,
+            total: 2
+          }
+        }
+      })
+    ).as("DataSourceOne");
+    cy.route(
+      "POST",
+      "/api/1.0/requests/data_sources/22",
+      JSON.stringify({
+        status: 200,
+        response: {
+          data: [],
+          meta: {
+            filter: "",
+            sort_by: "",
+            sort_order: "",
+            count: 2,
+            total_pages: 1,
+            current_page: 1,
+            from: 1,
+            last_page: 1,
+            path: "/api/1.0/collections/1/records",
+            per_page: 10,
+            to: 2,
+            total: 2
+          }
+        }
+      })
+    ).as("DataSourceTwo");
+  });
+
+  it("Metrics to get typing speed", () => {
+    cy.loadFromJson("RAOS_1.0.0_-_People_2.json", 7);
+
+    // set initial data to test the screen
+    cy.setPreviewDataInput(initialData);
+
+    // Wait Designer to load before continuing to Preview
+    cy.wait(5000);
+
+    cy.get("[data-cy=mode-preview]").click();
+
+    // set init screen test data
+    let logText;
+    let measure;
+    let t0;
+    let t1;
+    const delayValue = 50;
+    const text = "12345678901234567890123456789012345678901234567890";
+
+    // measure time to type 50ms delay
+    cy.get("[data-cy=preview-content] [name='middleName']")
+      .eq(0)
+      .then((element) => {
+        t0 = new Date().getTime();
+        return element;
+      })
+      .type(text, {
+        delay: delayValue
+      })
+      .then((element) => {
+        t1 = new Date().getTime();
+        measure = t1 - t0;
+        logText =
+          "Characters Number: " +
+          text.length +
+          " =>  Time: " +
+          measure +
+          "ms \n";
+        cy.log(logText).then(() => {
+          return element;
+        });
+        cy.writeFile("tests/e2e/fixtures/results.txt", logText, {
```

Comment:
`tests/e2e/fixtures` folder is not stored by circleci

You could try to add an additional store_path in `.circleci/config.yml` configuration file, like:
```
    - store_artifacts:
        path: tests/e2e/screenshots
    - store_artifacts:
        path: tests/e2e/metrics
```
see also: https://circleci.com/docs/configuration-reference/#storeartifacts

Or you could try store it to `tests/e2e/screenshots`

Reply:
If you want to store to `tests/e2e/metrics`
you must create the folder before run test, (in `.circleci/config.yml`):

```
    - run: mkdir tests/e2e/metrics
    - run:
        name: run tests
```